grpc/tests/src/test/system_contracts/auction/distribute.rs

```diff
@@ -415,7 +415,7 @@ fn should_distribute_delegation_rate_half() {
     let validator_1_balance =
         withdraw_validator_reward(&mut builder, *VALIDATOR_1_ADDR, VALIDATOR_1);
     let expected_validator_1_balance =
-        (expected_total_reward * validator_share + remainders).to_integer();
+        (expected_total_reward * validator_share + remainders - U512::from(2)).to_integer();
```

Comment:
Could we call this `rounded_amount`, defined above with `remainders`?

Reply:
Did this (and the rest as well in distribute.rs)